build.sh

```diff
@@ -18,6 +18,11 @@ if [[ "${SHOULD_BUILD}" == "yes" ]]; then
 
   cd vscode || { echo "'vscode' dir not found"; exit 1; }
 
+  if [[ "${VSCODE_ARCH}" == "ppc64le" ]]; then
+    export ELECTRON_SKIP_BINARY_DOWNLOAD="true"
```

Comment:
You can remove those lines since it's already in `prepare_vscode.sh` (just before `yarn --frozen-lockfile`)

Reply:
Removed